src/main/java/pl/javastart/movieclub/web/GenreController.java

```diff
@@ -22,15 +24,25 @@ public class GenreController {
     private final MovieService movieService;
 
     @GetMapping("/genre/{name}")
-    public String getGenre(@PathVariable String name, Model model){
```

Comment:
Zmienil byl sciezke w tym @GetMapping na "/{name}" a /genre dał do @RequestMapping nad nazwa klasy. Po drugie szukanie encji po nazwie to nie najlepszy pomysł. zmien lepiej na po id

Reply:
zrobione :)